index.js

```diff
@@ -1,27 +1,58 @@
 module.exports = {
-  root: true,
-  parser: "@typescript-eslint/parser",
-  parserOptions: {
-    project: "./tsconfig.json",
+  env: {
+    browser: true,
+    es2022: true,
+    node: true,
   },
-  settings: {
-    react: {
-      version: "17",
-    },
-  },
-  plugins: ["@typescript-eslint", "react", "react-hooks"],
   extends: [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:json/recommended",
     "plugin:react/recommended",
     "plugin:react/jsx-runtime",
     "plugin:react-hooks/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:typescript-sort-keys/recommended",
+  ],
+  ignorePatterns: ["build", "node_modules", "package-lock.json"],
```

Comment:
Isn't this `build` pattern assuming too much? What if the build is stored in `dist`?

Reply:
Yep, maybe it's better to let each project decide where the artifacts folder is located